src/Core/Registration/ColoredICP.cpp

```diff
@@ -233,13 +230,13 @@ double TransformationEstimationForColoredICP::ComputeRMSE(
 RegistrationResult RegistrationColoredICP(const PointCloud &source,
 		const PointCloud &target, double max_distance,
 		const Eigen::Matrix4d &init/* = Eigen::Matrix4d::Identity()*/,
-		const ICPConvergenceCriteria &criteria/* = ICPConvergenceCriteria()*/)
+		const ICPConvergenceCriteria &criteria/* = ICPConvergenceCriteria()*/,
+		double lambda_geometric/* = 0.968*/)
```

Comment:
I am not quite sure whether this extension is acceptable. I put ``lambda_geometric`` as the last entry of the function call in order to make users don't have to care about this parameter.